web/client/plugins/Swipe.jsx

```diff
@@ -0,0 +1,53 @@
+/*
+* Copyright 2020, GeoSolutions Sas.
+* All rights reserved.
+*
+* This source code is licensed under the BSD-style license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+import React from 'react';
+import { connect } from 'react-redux';
+import { createSelector } from 'reselect';
+
+import { getSelectedLayer, layerSwipeSettingsSelector } from '../selectors/layers';
+
+import { createPlugin } from '../utils/PluginsUtils';
+
+import SwipeSettings from './SwipeSettings';
+import XYSwipeSupport from '../components/map/openlayers/swipe/XYSwipeSupport';
```

Comment:
We will need some sort of wrapper that dynamically loads the required support for the current map type (OL only).
@allyoucanmap did you already do something in your experiments with support injected by pluguins? Here we are introducing and using the Map items injection via plugin-system